### ISSUE TITLE
SerializerGenerator throws exception when IsRecursive = true and target primitive types

### ISSUE THREAD

**Issue by neuecc** [msgpack/msgpack-cli#105]
New feature of ver 0.6.0 `SerializerCodeGenerationConfiguration.IsRecursive` is good.
But if target type contains primitive types, throws serialization exception.

```csharp
var settings = new SerializerCodeGenerationConfiguration
{
    IsRecursive = true,
};
var code = SerializerGenerator.GenerateCode(settings, typeof(int));
```

```csharp
System.Runtime.Serialization.SerializationException: Cannot serialize type 'System.Int32' because it does not have any serializable fields nor properties.
```

My expected result is do nothing.

**Comment by yfakariya** [comment]
Thank you for suggestion. It is bug of recursive target selection.

**Comment by yfakariya** [comment]
Fixed in 0.6.1.
